isso/utils/html.py

```diff
@@ -17,25 +17,17 @@ def allow_attribute_class(tag, name, value):
         return name == "class" and bool(Sanitizer.code_language_pattern.match(value))
 
     def __init__(self, elements, attributes):
-        # attributes found in Sundown's HTML serializer [1]
-        # - except for <img> tag, because images are not generated anyways.
-        # - sub and sup added
-        #
-        # [1] https://github.com/vmg/sundown/blob/master/html/html.c
-        self.elements = ["a", "p", "hr", "br", "ol", "ul", "li",
-                         "pre", "code", "blockquote",
-                         "del", "ins", "strong", "em",
-                         "h1", "h2", "h3", "h4", "h5", "h6", "sub", "sup",
-                         "table", "thead", "tbody", "th", "td"] + elements
+        self.elements = elements
 
         # allowed attributes for tags
         self.attributes = {
-            "table": ["align"],
-            "a": ["href"],
-            "code": Sanitizer.allow_attribute_class,
             "*": attributes
         }
 
+        # If "code" elements are allowed, allow "language-*" CSS classes for syntax highlighting
```

Comment:
Could you add a few more quality-of-life improvements to `Markup` in a separate commit below?

E.g. `if highlight` -> `allow <mark>`

Reply:
I added checks for the `highlight` and `underline` extensions.